Readme.md

```diff
@@ -40,6 +40,7 @@ The `options` argument allows you to customize the client with the following pro
 - endpoint: to override the SOAP service's host specified in the `.wsdl` file.
 - request: to override the [request](https://github.com/request/request) module.
 - httpClient: to provide your own http client that implements `request(rurl, data, callback, exheaders, exoptions)`.
+- forceSoap12: to set proper headers for SOAP v1.2
```

Comment:
in a lot of samples tests we got `xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/"` or `xmlns:SOAP="http://schemas.xmlsoap.org/wsdl/soap12/"` SOAP v1.2 namespaces in WSDL, but in request we got SOAP v1.1 namespace `xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"`, [GetAccountXml](https://github.com/vpulim/node-soap/tree/e4bfa93f159b8ad10af696967c39b1eb8ade76a3/test/request-response-samples/GetAccountXml__non_xmlns_attributes_should_be_returned) for example
it looks very confusing, and I don't know a proper way to determine SOAP version from WSDL